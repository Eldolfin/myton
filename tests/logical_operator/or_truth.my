# False and nil are False.
print False or "ok" # expect: ok
print None or "ok" # expect: ok

# Everything else is True.
print True or "ok" # expect: True
print 0 or "ok" # expect: "ok"
print "s" or "ok" # expect: s
