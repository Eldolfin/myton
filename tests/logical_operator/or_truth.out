ok
ok
True
ok
s
